plugins/flytekit-flyin/flytekitplugins/flyin/vscode_lib/decorator.py

```diff
@@ -124,7 +106,7 @@ def download_file(url, target_dir: Optional[str] = "."):
     return local_file_name
 
 
-def download_vscode(vscode_config: VscodeConfig):
+def download_vscode(vscode_config: Optional[VscodeConfig] = None):
```

Comment:
it cannot be None because if it is none, vscode decorator creates a default vscodeconfig.